jira/github_jira.py

```diff
@@ -28,13 +28,21 @@
                'redhat-cop/cloud.aws_troubleshooting',
                'redhat-cop/cloud.azure_ops',
                'redhat-cop/cloud.gcp_ops',
+               'redhat-cop/cloud.terraform_ops',
 ]
 
 g = Github(config['gh_token'])
-#jiraconn = JIRA(token_auth=config[jira_token']), server=config['jira_server'])
 jiraconn = JIRA(token_auth=config['jira_token'], server=config['jira_server'])
 
-jira_issues = jiraconn.search_issues('project=ACA and labels=github')
+jira_issues =  []
```

Comment:
Looks like I have local changes that I haven't pushed to GH yet.  The jira API has a maxResults option:
`jira_issues = jiraconn.search_issues('project=ACA and labels=github', maxResults=1000)`